test/python_client_test.py

```diff
@@ -103,43 +104,61 @@ def test_success_get_srv_keyspace(self):
     with self.assertRaises(ValueError):
       small.keyspace_id_to_shard_name_for_db_type(0x6000000000000000, 'replica')
 
+  KEYSPACE_ID_80 = struct.Struct('!Q').pack(0x80)
```

Comment:
This 0x80 is a bit confusing, I think you meant 0x8000000000 instead?

0x80 would be one of the first values in the 00-80 range for instance, not the middle value as one would expect when you see 0x80 like this...